portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro

```diff
@@ -477,6 +477,28 @@ definition {
 			expected = "${editStructuredContentId}");
 	}
 
+	macro assertStructuredContentTitleIsExpectedWithCorrectOrder {
+		Variables.assertDefined(parameterList = "${responseToParse}");
+
+		var structuredContentIdActual = JSONUtil.getWithJSONPath("${responseToParse}", "$..title");
```

Comment:
this should be structuredContentTitleExpected = "Web Content 3,Web Content 1,Web Content 2";